anchore_engine/services/policy_engine/engine/vulns/mappers.py

```diff
@@ -401,8 +433,12 @@ def to_engine(
     "busybox": DistroMapper(
         engine_distro="busybox", grype_os="busybox", grype_like_os=""
     ),
+    "windows": DistroMapper(
+        engine_distro="windows", grype_os="windows", grype_like_os=""
+    ),
 }
 
+# TODO: this does not appear to be used and should probably be removed.
```

Comment:
This was initially added to support stateless/inline scanning of syft sbom. Should be safe to remove it along with the unit tests. If you decide to leave it here, add the mapping for microsoft to maintain the symmetry

Reply:
Will remove it in that case. Thank you!